doc/manual/command-ref/conf-file.xml

```diff
@@ -656,6 +656,72 @@ password <replaceable>my-password</replaceable>
 
   </varlistentry>
 
+  <varlistentry xml:id="conf-post-build-hook">
+    <term><literal>post-build-hook</literal></term>
+    <listitem>
+      <para>Optional. The path to a program which is executed after
+      each build.</para>
+
+      <para>This option can only be set in the nix.conf, or by trusted
```

Comment:
`the nix.conf` -> `<filename>nix.conf</filename>`.

Reply:
Fixed, thanks